firebase/manifests/android/build.gradle

```diff
@@ -1,6 +1,6 @@
 
 dependencies {
-    compile 'com.google.firebase:firebase-analytics:17.5.0'
-    compile 'com.google.android.gms:play-services-base:17.4.0'
+    implementation platform('com.google.firebase:firebase-bom:26.5.0')
```

Comment:
Are you sure this is correct? I'm getting a crash with the example packaged with the extension:

```
firebase  E  Unable to check Google Play services availablity as the com.google.android.gms.common.Google
			 ApiAvailability class is not present in this application.
```

The dependency map shows this:

https://github.com/firebase/firebase-cpp-sdk/blob/a7e77482bea5ac5dfa646bd4a3bfc8a608bbb786/Android/firebase_dependencies.gradle#L23-L24

Reply:
Yes, you are right. I missed nessessary dependecy.